### PR TITLE
chore(website): add categorization blog post

### PR DESCRIPTION
**Pull request by nkrantz**
- Added stage categorization blog post
- Added avatar photo for Aayush's article
- Updated Readme.md for clearer instructions to add an article
- Converted Shadi's post from internal to external ([context](https://github.com/twilio-labs/paste/pull/1656#issuecomment-879452882))

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Review by gloriliale — APPROVED** [review]
Looks great!!

File: packages/paste-website/src/pages/articles/README [hunk 1]
```diff
@@ -13,6 +13,8 @@ All articles are MDX pages. Like all gatsby pages, the url is reflective of the
 
 File names are in the format `YYYY-MM-DD-title-of-post.mdx`
 
+Create a new file in the `/articles` directory and copy/paste an existing article's file contents.
```
Comment (TheSisb) [inline packages/paste-website/src/pages/articles/README:16]: Nice improvements here 👍

**Review by TheSisb — APPROVED** [review]

File: packages/paste-website/src/pages/articles/README [hunk 2]
```diff
@@ -21,7 +23,7 @@ Each file must start with a Frontmatter section that controls the meta data of t
 
 - **Title** - Title of the article or post
 - **date** - `YYYY-MM-DD` Date at which it was published
-- **slug** - URL slug that the full article appears at, absolute to the root of the website domain
+- **slug** - URL slug that the full article appears at, absolute to the root of the website domain (must end with a `/`)
```
Comment (loreina) [inline packages/paste-website/src/pages/articles/README:26]: curious to learn: why should the URL slug end with `/`?

File: packages/paste-website/src/pages/articles/2021-07-21-pastes-path-to-a-transparent-package-categorization-system.mdx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  authorLink={props.data.mdx.frontmatter.author_link}
+/>
+
+---
+
+<contentwrapper>
+
+<ArticleContent>
+
+_When integrating a library into a project, you’ll likely want to ensure that it’s a mature and well-maintained library that won’t be deprecated a month down the line. If your project is a customer-facing platform and the library maintains your frontend components, all the more reason to have trust. Transparency between the library and the consumer is integral to building that trust._
```
Comment (loreina) [inline packages/paste-website/src/pages/articles/2021-07-21-pastes-path-to-a-transparent-package-categorization-system.mdx:74]: vvv small nit, the italics create nice visual hierarchy at the top but not sure if it's a quote or summary of something later in the post

**Review by loreina — COMMENTED** [review]
loved reading this!

File: packages/paste-website/src/pages/articles/README (hunk 2 above)
Comment (nkrantz) [reply packages/paste-website/src/pages/articles/README:26]: I'm not sure, but my page wasn't loading before I remembered to add it 😅 

@SiTaggart why does the URL always end in a `/`?

File: packages/paste-website/src/pages/articles/2021-07-21-pastes-path-to-a-transparent-package-categorization-system.mdx (hunk 3 above)
Comment (nkrantz) [reply packages/paste-website/src/pages/articles/2021-07-21-pastes-path-to-a-transparent-package-categorization-system.mdx]: removed italics!

File: packages/paste-website/src/pages/articles/README (hunk 2 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/articles/README:26]: 🤷🏼 Gatsby thing. Been bitten by it many a time

**Pull request merged**